hasher-matcher-actioner/webapp/src/pages/MatchIndex.jsx

```diff
@@ -0,0 +1,251 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+ */
+
+import React, {useState, useEffect} from 'react';
+import {useHistory} from 'react-router-dom';
+import {PropTypes} from 'prop-types';
+import {
+  Container,
+  Col,
+  Button,
+  Row,
+  Collapse,
+  InputGroup,
+  Form,
+  FormControl,
+} from 'react-bootstrap';
+import Spinner from 'react-bootstrap/Spinner';
+import classNames from 'classnames';
+
+import {
+  fetchAllMatches,
+  fetchMatchesFromContent,
+  fetchMatchesFromSignal,
+} from '../Api';
+import {timeAgo} from '../utils/DateTimeUtils';
+import ContentMatchPane from '../components/ContentMatchPane';
+import {useQuery} from '../utils/QueryParams';
+
+export default function MatchIndex() {
```

Comment:
I think I get the divination of `MatchIndex`... it's like index.html, right? Take with a grain of salt given my time with React is not huge, but in WWW I think you often see something that matches the route or the main "entity" of the "page" (e.g. `<Matches>`). I don't ever remember seeing the word Index involved with a "root" component for a page.